airflow/operators/python.py

```diff
@@ -171,7 +171,7 @@ def execute(self, context: Dict):
         self.op_kwargs = determine_kwargs(self.python_callable, self.op_args, context)
 
         return_value = self.execute_callable()
-        self.log.info("Done. Returned value was: %s", return_value)
+        self.log.debug("Done. Returned value was: %s", return_value)
```

Comment:
I do not think, changing the log level is good solution. There are many use cases where peopel might rely on the return value appear in logs.

I think better one might be to add optional parameter (default to True) `show_return_value_in_logs` that could be set to False tu suppress printing the value to the logs - with appropriate explanation of the "big" values.